### ISSUE TITLE
Multiline comments are squashed without whitespace

### ISSUE THREAD

**Issue by Alxandr** [dotansimha/graphql-code-generator#950]
**Describe the bug**
The following gql:

```gql
  # Decides the order a module will run in. Modules with the same
  # level, will run in parallel.
  level: Int!
```

turns into this in typescript

```ts
  /** Decides the order a module will run in. Modules with the samelevel, will run in parallel. */
  level: number;
```

Notice that `same` and `level` get's merged to `samelevel`.

**Expected behavior**

I expected the comment to either be multiline, or be

```ts
/** Decides the order a module will run in. Modules with the same level, will run in parallel. */
```

**Environment:**
 - OS: OSX
 - Codegen: v0.14.4
 - Node: v11.2.0

**Additional context**
I've looked into the code a bit, and in all likelihood, changing `oneLineTrim` to `oneLine` will fix this issue:

https://github.com/dotansimha/graphql-code-generator/blob/master/packages/utils/plugins-handlebars-helpers/src/index.ts#L2

**Comment by kamilkisiela** [comment]
@Alxandr Thanks :) I will take a look at it at the end of the week but maybe you're up to fix it in a PR? I could help

**Comment by kamilkisiela** [comment]
Released in `v0.15.0`
